app/controllers/api/v1/visualization/annotations_controller.rb

```diff
@@ -330,6 +340,9 @@ def facets
             response 200 do
               key :description, '2-column TSV of column header names from the gene list file.  Column headers are NAME (the cell name) and the name of the gene list'
             end
+            response 400 do
```

Comment:
Is there an easy way to abstract this duplicative code?  It'd help denoise.  If not, that's fine!

Reply:
Yeah there is, I was just being lazy...